plugins/arOaiPlugin/lib/arOaiPluginComponent.class.php

```diff
@@ -96,7 +96,7 @@ public function setUpdateParametersFromRequest($request)
     }
   }
 
-  public function getUpdates()
+  public function getUpdates($filterDrafts = false)
```

Comment:
Can you use the `array $options` pattern? I'm not doing CR of this pull request but I ran into this accidentally.